presto-docs/src/main/sphinx/develop/functions.rst

```diff
@@ -282,3 +282,25 @@ function follows:
   is used when performing a ``GROUP BY`` aggregation, and an implementation
   will be automatically generated for you, if you don't specify a
   ``AccumulatorStateFactory``
+
+Deprecated Function
+-------------------
+
+The ``@Deprecated`` annotation can be used to mark a function as deprecated
+and should no longer be used. Presto will generates a warning whenever a SQL
+uses a deprecated function. When a function is deprecated, it should also be
```

Comment:
When a functions is deprecated, the ``@Description`` needs to be replaced with a note about the deprecation and the replacement function: